SRC/css-grids/css-grids.css

```diff
@@ -0,0 +1,59 @@
+.mainContainer{
+    background-image: url(https://cdn.cjr.org/wp-content/uploads/2019/07/AdobeStock_100000042-e1563305717660-686x371.jpeg);
+     display: grid;
+    grid-template-columns: 33% 33% 33%; 
+    grid-template-rows: 2fr 3fr 1fr;
+   /* grid-template-areas:  
+    "..... .welcome ......."
+    ".calculator .... .todo"
+    ".. .data .clock "; */
+}
+h1{
+    color: rgb(248, 223, 142);
+}
+section{
+    border: 2px solid whitesmoke;
+    background: whitesmoke transparent
```

Comment:
you can only add one color value to background so this won't work. if you wanted to make a gradient with multiple colors you'll want to use the `background: linear-gradient();` property and value. 
check out: [cssgradient.io](https://cssgradient.io/)